src/SampleExtension/Extension.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Dynamo.Extensions;
+using Dynamo.Graph.Nodes;
+using Dynamo.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SampleExtension
+{
+    public class Extension : IExtension
```

Comment:
Can we add a short description here about extensions similar to [this](https://github.com/DynamoDS/DynamoSamples/blob/master/src/SampleViewExtension/SampleViewExtension.cs#L9) one in the view extension sample